BackEndApp/src/main/java/com/example/backendapp/Controller/PlantController.java

```diff
@@ -0,0 +1,21 @@
+package com.example.backendapp.Controller;
+
+import com.example.backendapp.Model.Plant;
+import com.example.backendapp.Service.PlantService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+public class PlantController {
+
+    private final PlantService plantService;
+
+    @CrossOrigin(origins = "http://localhost:3000")

```

Comment:
You can add @CrossOrigin annotation at class level. Do the same for the other java files.